CommandHelper/LocalPackages/commands/chars/wildmutt.yml

```diff
@@ -0,0 +1,37 @@
+buffs:
+- /effect <plr> 15 9999 1 true
+- /effect <plr> 16 9999 1 true
+- /effect <plr> 5 9999 5 true
+- /give <plr> minecraft:iron_bars 2 16:10 name:&6Wildmutt's_Claws
+- /mgive <plr> trackingwand
```

Comment:
buffs have been split into 2 sections now, effects and items. please see superman.yml for reference